src/Horus/Global.hs

```diff
@@ -359,8 +364,22 @@ solveContract = do
     )
       infos
  where
-  isStandardSource :: Set ScopedFunction -> ScopedFunction -> Bool
-  isStandardSource inlinables f = f `notElem` inlinables && not (isWrapper f)
+  userAnnotatedSources :: Set ScopedFunction -> [LabeledInst] -> GlobalL (Set ScopedFunction)
+  userAnnotatedSources inlinableFs rows =
+    getProgram >>= \prog ->
+      let functionsWithBodies = functionsOf rows prog
```

Comment:
There is a bug here that triggers fromJust under some circumstances unknown to me. This needs investigation. Preferably, this whole bit of code wouldn't be here in the first place because we're replicating this kind of logic in FunctionAnalysis.hs and frankly even in Module.hs.